src/commands/package.yml

```diff
@@ -22,4 +26,4 @@ steps:
           -e SKPR_USERNAME=${SKPR_USERNAME} \
           -e SKPR_PASSWORD=${SKPR_PASSWORD} \
           <<parameters.docker_image>> \
-          skpr package <<parameters.version>> --print-manifest > skpr-manifest.json
+          skpr package <<parameters.version>> <<^parameters.push>>--no-push<</parameters.push>> --print-manifest > skpr-manifest.json
```

Comment:
Is this a special CircleCI syntax?

Reply:
Truthy/falsy checks which are practically undocumented by CircleCI.
https://discuss.circleci.com/t/if-else-parameter-syntax/42815/2

Very handy when not abstracting to a script actually.